ocs_ci/deployment/cnv.py

```
@@ -385,6 +385,43 @@ def post_install_verification(self, raise_exception=False):
+        logger.info(
+            f"Waiting up to {timeout}s for HyperConverged systemHealthStatus to become 'healthy'"
+        )
+        ocp = OCP(kind=constants.HYPERCONVERGED, namespace=self.namespace)
+        for sample in TimeoutSampler(
+            timeout=timeout,
+            sleep=sleep,
+            func=lambda: ocp.get(resource_name=constants.KUBEVIRT_HYPERCONVERGED)
+            .get("status", {})
+            .get("systemHealthStatus"),
+        ):
+            logger.info(f"HyperConverged systemHealthStatus: {sample}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)